nima_core/storage/process_guard.py

```
@@ -193,6 +336,41 @@ def find_zombie_gateways() -> list:
+
+    try:
+        pid = int(PID_FILE.read_text().strip())
+        # Check if process is actually running
+        try:
+            os.kill(pid, 0)
+            # Process is running - check if it's actually a gateway
+            try:
+                cmdline = Path(f"/proc/{pid}/cmdline").read_text().replace('\0', ' ')
+                if 'openclaw' in cmdline or 'gateway' in cmdline:
+                    return []  # Not a zombie - active gateway
+            except Exception as exc:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)